msteams-platform/m365-apps/specify-runtime-requirements.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Specify Microsoft 365 host runtime requirements in your app manifest
+description: Specify your app's runtime requirements in app manifest to provide a deliberate experience on supported Microsoft 365 hosts. 
```

Comment:
@erikadoyle, we'd include more keywords in the meta description. Here's a suggestion:

```suggestion
description: Specify runtime requirements, such as one-way or mutual dependencies and capability requirements, in app manifest to tailor app experience in Microsoft 365.
```

Reply:
How about:

Specify app runtime requirements, such as one-way or mutual dependencies and capability requirements, to tailor its user experience in different Microsoft 365 host applications.

Reply:
@erikadoyle, I've reduced the description you've suggested to <160 chars. Let me know what you think.